### ISSUE TITLE
create Postgresql script for init db

### ISSUE THREAD

**Issue by benjaminW78** [benjaminW78/pokeGoNetwork#11]
Set postgresql initialization script for this project and play it after git clone.

**Comment by november-eleven** [comment]
Nope, use a migration tool like [Migrate](https://github.com/mattes/migrate), [Flyway](https://flywaydb.org/) or [Evolution](https://www.playframework.com/documentation/2.5.x/Evolutions), but for nodejs.

**Comment by benjaminW78** [comment]
@aymericbeaumet  you know this better than me. what so you think about using one of this solutions ?

**Comment by aymericbeaumet** [comment]
don't think you need to over-engineer here, writing simple js scripts to populate the db might be enough for your needs.

which orm do you plan to use? I recommend http://docs.sequelizejs.com/en/latest/

**Comment by aymericbeaumet** [comment]
it will take care of creating the table if it's missing from the database

**Comment by benjaminW78** [comment]
woh this look very convenient to me !
